Orange/widgets/data/owpythonscript.py

```diff
@@ -645,7 +667,7 @@ def saveScript(self):
             f.close()
 
     def initial_locals_state(self):
-        d = dict([(i.name, getattr(self, i.name, None)) for i in self.inputs])
+        d = dict([(name, getattr(self, name, None)) for name in self.inputs_names])
```

Comment:
Actually not, I just changed the (uncommented) code for `DistanceMatrix`. You can keep it commented.